package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "scripts": {
     "start": "./bin/server.js",
-    "test": "jshint *.js **/*.js && jscs *.js **/*.js && nodeunit"
+    "test": "jshint {.,bin,lib,lib/repo,test}/*.js && jscs {.,bin,lib,lib/repo,test}/*.js && nodeunit"
```

Comment:
So without `excludeFiles`, `node_modules` is checked, but with an empty option, its excluded?

The docs say "`node_modules` folder is excluded by default", that doesn't clarify it.